### ISSUE TITLE
EventAggregator is a plugin that is not documented in the "Plugins" section

### ISSUE THREAD

**Issue by dkent600** [aurelia/documentation#339]
EventAggregator is a plugin that is not documented in the "Plugins" section.  Seems like it should be.

**Comment by Alexander-Taran** [comment]
it's a pub sub feature 
although naming it a plugin in cheat-sheet might be confusing
http://aurelia.io/docs/fundamentals/cheat-sheet#the-event-aggregator

**Comment by dkent600** [comment]
I don't think this kind of topic belongs in the cheat sheet which I think should be no more than a set of code snippets for doing common and handy things.

One can see `EventAggregator` here referred-to as a plugin:

http://aurelia.io/docs/api/event-aggregator/function/configure

It is initialized by `standardConfiguration`:  http://aurelia.io/docs/api/framework/class/FrameworkConfiguration/method/standardConfiguration

But from reading the `EventAggregator` documentation one would never know how otherwise to initialize it.

I'll stick by my assertion that this topic should be under "Plugins".

**Comment by Alexander-Taran** [comment]
Well.. I hope you'll get more time soon and be able to provide a PR for this..
I don't see how with 200+ issues tagged 'documentation' the team can do it on their own.

Oh, and would you be so kind to review all your open issues and close those that are not relevant any more. I've noticed you closed some after my comments.
It would be very nice.
